packages/dev/inspector/src/inspector.ts

```diff
@@ -388,6 +388,14 @@ export class Inspector {
             ...userOptions,
         };
 
+        // load the font, unless asked to skip it
+        if (!options.skipDefaultFontLoading || (globalThis as any)?.BABYLON_SKIP_FONT_LOADING !== true) {
+            const font = document.createElement("link");
+            font.rel = "stylesheet";
+            font.href = "https://use.typekit.net/cta4xsb.css";

```

Comment:
Just to be sure, it's expected you use a different url than the one we currently use (`https://use.typekit.net/xfi0rwe.css`)?

Reply:
Yes, they are essentially the same. We use https://use.typekit.net/cta4xsb.css in the tools, which has more font definitions. they are not loaded until the browser uses them.